technical-committee/shared-development-roadmap/introduction.md

```diff
@@ -0,0 +1,129 @@
+term "CLARIAH+ epic" now.
+
+## Goals
+
+In this phase 2 we will put the emphasis on formulating high-level epics and
+the underlying services, sofware and data components that implement these
+epics.
+
+The goals for phase 2 are:
+
+* Establish which are the *core shared* CLARIAH+ epics. Emphasis for the shared
+ development roadmap is on these core shared stories, i.e. we are trying to steer the focus for the remainder of
+ CLARIAH+ to development of core infrastructural components.
+* Define for each:
+    1. What it entails; what are the user stories that make up the epic? Are we all on board with the plan?
+    2. Which software/service/data components are needed and what interoperability standards will be adhered to?
+    3. How do the components fit together to make a whole? How do the core components relate to other components? By
+       definition, the core shared components should be heavily inter-related. The use of diagrams is strongly encouraged here.
+    4. Where can service instance(s) be hosted?
+    5. What people need/want to be involved?
+    6. Come up with a resource planning that can serve as input for the revised 2022-2023 workplan. For each
+       epic we have a dedicated GitHub projects kanban board and list that should be used for planning and during
+       development (this offers transparency and facilitates cross-institutional collaboration).
+    7. Come up with arguments for additional investments from CLARIAH in the services if needed
+* A similar procedure applies to the domain-specific services, though the details of these may often be confined to a
+  particular WP. We do need broad agreement which domain-specific services we consider worth the effort and which to drop.
+
+The overall objectives of the Shared Development Roadmap are:
+
+* Identify and foster cross-WP and cross-institutional collaboration opportunities (CLARIAH services spanning components from multiple WPs)
+* Fill 'gaps' needed to provide an actual common shared infrastructure.
+* Get an accurate overview of how CLARIAH epics relate to everything that has been developed in CLARIAH.
+* We seek to harmonize various solutions developed within CLARIAH; determine which are mature and have potential, which can be discarded, which to go forward with in a potential successor project.
+* Foster interoperability between software/data components
+
+## Definitions
+
+### CLARIAH Epic
+
+A **CLARIAH Epic** is describes the needs of a scholar in broad and generic themes. The epic is made up of multiple user
+stories, which are mostly described from a scholarly perspective. The epic effectively formulates a 'CLARIAH
+service' in the largest sense of the word; a product/functionality offered by CLARIAH to scholars.
+
+The implementation(s) of an epic facilitate a certain *scholarly workflow*; a scholarly workflow is defined as a
+sequence of steps (e.g. analysis, data transformation, presentation) that serves specific needs of a scholar.
```

Comment:
And this throws ‘workflow’ in the mix (next to ‘epic’ and ‘service’), adding to the confusion.

Reply:
agreed, I think we can remove that too, we speak of 'components' later on anyways